cirq/google/optimizers/convert_to_sqrt_iswap_test.py

```diff
@@ -0,0 +1,121 @@
+from typing import cast, List, Tuple
+import numpy as np
+import pytest
+import sympy
+
+import cirq
+import cirq.google.optimizers.convert_to_sqrt_iswap as cgoc
+import cirq.google as cig
+
+
+def _unitaries_allclose(circuit1, circuit2):
+    unitary1 = cirq.unitary(circuit1)
+    unitary2 = cirq.unitary(circuit2)
+    if unitary2.size == 1:
+        # Resize the unitary of empty circuits to be 4x4 for 2q gates
+        unitary2 = unitary2 * np.eye(unitary1.shape[0])
+    return cirq.allclose_up_to_global_phase(unitary1, unitary2)
+
+
+@pytest.mark.parametrize(
+    'gate, expected_length',
+    [
+        (cast(cirq.Gate, cirq.ISWAP), 7),  # cast is for fixing mypy confusion
+        (cirq.CZ, 8),
+        (cirq.SWAP, 13),
+        (cirq.CNOT, 9),
+        (cirq.ISWAP**0.5, 1),
+        (cirq.ISWAP**-0.5, 1),
+        (cirq.ISwapPowGate(exponent=0.5), 1),
+        (cirq.ISwapPowGate(exponent=-0.5), 1),
+        (cirq.FSimGate(theta=np.pi / 4, phi=0), 1),
+    ] +
+    [(cirq.SwapPowGate(exponent=a), 13) for a in np.linspace(0, 2 * np.pi, 20)]
```

Comment:
nit: instead of concatenating lists, might be cleaner to splat iterables into a single list:
```python
@pytest.mark.parametrize(
    'gate, expected_length',
    [
        (cast(cirq.Gate, cirq.ISWAP), 7),
        ...
        (cirq.FSimGate(theta=np.pi / 4, phi=0), 1),
        *[(cirq.CZPowGate(exponent=a), 8) for a in np.linspace(0, 2 * np.pi, 20)],
        *[(cirq.ISwapPowGate(exponent=a), 5) for a in np.linspace(0, 2 * np.pi, 20)],
        ...
    ])
```
I have no idea which of these the formatter will handle better, so feel free to ignore if this doesn't help.

Reply:
Good idea.  It looks a lot better.